commands/gban.js

```diff
@@ -34,7 +40,9 @@ exports.run = async (client, message, args) => {
 
     await client.db.updateUser(user.data)
 
-    message.channel.send(`${greentick} Banned \`${user.id}\` from ${potentialGuilds.size} servers`)
+    if (completemessage)
```

Comment:
This `if` statement is useless.

Reply:
No it isn't. Otherwise, if completemessage isn't defined, it'll have an error of 'cannot have function edit of undefined'